### PR TITLE
update contibuting

### PR DESCRIPTION
**Pull request by simonschwartz**
@angiegove @catherineedwards 

I have updated the contributing guide to include some more information about how to submit pull requests.

I also reframed some of the test criteria in the original contributing to be more guidelines for wriitng content rather than criteria we test you on.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -1,26 +1,80 @@
...
-- Submit a pull request.
-- The content in the pull request will be tested by the cloud.gov.au team against the criteria mentioned above.
+All the content for cloud.gov.au can be found in the `docs` folder. The content is written in [Markdown](https://daringfireball.net/projects/markdown/syntax). and organised into folders that replicate the Information Architecture structure of the site.
 
-## Requesting content
-If there is a particular piece of documentation that is missing, but you don't know how create it, raise a Github issue in the `cga_docs` repository requesting the content that you need.
+It's really important that we make sure our documentation is clear, concise and written in plain english where possible. Here are some tools we use to help us do this:
+
+- [Hemmingway editor](http://www.hemingwayapp.com/) - Simple app that checks your content for readability
+- [Digital Transformation Office content styleguide](http://content-style-guide.apps.staging.digital.gov.au/)
```
Comment (tessereth) [inline contributing.md]: We're the DTA now right?

File: contributing.md [hunk 2]
```diff
@@ -1,26 +1,80 @@
...
+Simply edit the relevant markdown file and send a pull request. If you are new to pull requests you can read our [pull request guide](#submitting-a-pull-request)
+
+## Code contributions
+
+We welcome any pull requests to fix issues and add features to docs.cloud.gov.au. Here are a
+few tips about how the project is setup.
+
+### Updating the theme
+
+We have html, css and JS are packaged as part of a MKDocs UI Kit theme. If you would like to submit a
```
Comment (tessereth) [inline contributing.md]: Odd grammar here. Possibly change `We have` to `Our`?

**Review by tessereth — COMMENTED** [review]
This reads really well. Thanks @simonschwartz!

**Review by angiegove — APPROVED** [review]
This is great 👍 , Thanks @simonschwartz
